src/components/ui/Box.tsx

```diff
@@ -71,7 +71,7 @@ export const Box: React.FC<BoxProps> = ({
   return (
     <div
       className={`relative ${className}`}
-      onMouseEnter={() => setIsHover(!isMobile)}
+      onMouseEnter={() => setIsHover(true)}
```

Comment:
`setIsHover(!mobile)` was introduced in another PR to solve the following issue on mobile:

When players click the 2nd/3rd item in the inventory shortcut, it stays "selected" (or appears to be):
![image](https://user-images.githubusercontent.com/107602352/196023495-0120695c-7f3e-435c-918a-c8ca9801a22d.png)

This is because in line 154 (of the same file), the selected indicator is also shown when `isHover` is true.  On mobile, when you tap the 2nd/3rd item, `isHover` is still true for the 2nd/3rd item because the last touch is still at that area.

The previous PR attempted to fix this issue by `setIsHover(!mobile)`  instead of `setIsHover(true)`, so on mobile `isHover` is always false.  This created another issue where players are not able to see the full string (eg. 1306.429 potatoes instead of 1.3k potatoes) on mobile at all, because on line 144 `isHover` is always false for mobile.  To address this issue, `setIsHover(!mobile)` is reverted back to `setIsHover(true)`, while the selected indicator is not controlled by `isHover` anymore on moblie (changes in line 154 for this PR)

@AzaroQL also discovered that `setIsHover(!mobile)` is affecting his environment on a touchscreen computer, thus the changes are made in `[useIsMobile.ts`.